kube/pod.go

```diff
@@ -509,12 +509,28 @@ func getEnvVarsFromConfigs(configs model.Variables, settings ExportSettings) (he
 	}
 
 	fieldRef := helm.NewMapping("fieldPath", "metadata.namespace")
-
 	envVar := helm.NewMapping("name", "KUBERNETES_NAMESPACE")
 	envVar.Add("valueFrom", helm.NewMapping("fieldRef", fieldRef))
-
 	env = append(env, envVar)
 
+	if settings.CreateHelmChart {
```

Comment:
Wouldn't this magic make more sense living near the `KUBE_SECRETS_GENERATION_COUNTER` &c stuff above, to keep it all in the same area?

Reply:
Checking ...

Reply:
@mook-as The `COUNTER` variable, etc. is in a loop.
The `NPROC` setup happens after that loop.
These are definitely distinct areas.
I don't believe I can merge this, or that it would make sense to move `NPROC` into the loop. They are not part of the `configs` the loop iterates over.

IMHO nothing to do.